js/tex.jsx

```diff
@@ -143,6 +191,12 @@ var TeX = React.createClass({
                 jax.Remove();
             }
         }
+    },
+
+    emptyNode: function(node) {
```

Comment:
For some reason we were using jQuery just to empty out nodes. I made a little method to remove that dependency.